pkg/pillar/types/types.go

```diff
@@ -267,3 +267,19 @@ func FormatTriState(state TriState) string {
 	}
 	return ""
 }
+
+// UuidsToNum used for appNum on network instance
+type UuidsToNum struct {
```

Comment:
My previous comment about the extra 's' being to subtle and it makes sense to name things 'uuidpair*' instead of "uuids*' was intended as a global comment; this struct, all of the functions, etc.

Reply:
Sure. will do.